cupy/manipulation/add_remove.py

```diff
@@ -7,10 +10,21 @@
 # TODO(okuta): Implement insert
 
 
-# TODO(okuta): Implement append
+def append(arr, values, axis=None):
```

Comment:
NumPy accepts the following code.
```
>>> numpy.append(100, [0, 1, 2])
array([100,   0,   1,   2])
```

Reply:
How about the following implementation?
```py
def append(arr, values, axis=None):
    arr = cupy.asarray(arr)
    values = cupy.asarray(values)
    if axis is None:
        return core.concatenate_method((arr, values), 0).ravel()  # to avoid redundant copying
    else:
        return core.concatenate_method((arr, values), axis)
```